project.php

```diff
@@ -37,22 +45,28 @@
 <header class="p-3 bg-teal-50 flex justify-between">
     <h1 class="sm:text-5xl text-4xl"><a href="index.php">Project Manager</a></h1>
     <div class="pr-3 flex">
-        <a href=<?php echo "project.php?project=$idLink"?> class="p-3 bg-slate-300 rounded-l-lg border-y border-l">🇬🇧</a>
+        <?php
+        if ($pageLocale == 'uk') {
+            echo "<a href = 'project.php?project=$idLink&location=uk' class='p-3 bg-slate-300 rounded-l-lg border-y border-l' > 🇬🇧</a >";
+            echo "<a href = 'project.php?project=$idLink&location=us' class='p-3 rounded-r-lg border-y border-r' > 🇺🇸</a >";
+        } else {
+            echo "<a href = 'project.php?project=$idLink&location=uk' class='p-3  rounded-l-lg border-y border-l' > 🇬🇧</a >";
+            echo "<a href = 'project.php?project=$idLink&location=us' class='p-3 bg-slate-300 rounded-r-lg border-y border-r' > 🇺🇸</a >";
+        }
+        ?>
     </div>
 </header>
 <main class="p-3">
-
-
-            <div class="flex justify-between mb-3">
-                <h2 class="text-4xl font-bold mb-2"><?php echo $projectTitle.$projectDeadline ?>
-                    <a href="index.php" class="text-base text-blue-600 hover:underline ms-3">Return to all projects</a>
-                </h2>
-                <div class="flex items-center gap-3">
-                    <h3 class="text-3xl font-bold"><?php echo $clientTitle ?></h3>
-                    <img class="w-[50px]" src=<?php echo $client->logo ?> alt="client logo" />
-                </div>
-            </div>
-        <section class="flex gap-5 flex-nowrap h-[70vh] pb-5 overflow-x-auto">
+    <div class="flex justify-between mb-3">
+        <h2 class="text-4xl font-bold mb-2"><?php echo $projectTitle.$projectDeadline ?>
```

Comment:
There's a shorthand syntax for this. Instead of:

```
<?php echo $projectTitle.$projectDeadline ?>
```

You can just do this:

```
<?= $projectTitle.$projectDeadline ?>
```

Reply:
Try just replacing it in this file, don't worry about the whole project